<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>verificationpage</class>
 <widget class="QMainWindow" name="verificationpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>553</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Nirmala UI</family>
    <pointsize>-1</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
image: url(:/pishvazpic.JPG);
background-color: rgb(192, 221, 190);
font: 40px
}

#lineEdit_4{
font: bold 15px;
border-radius: 8px;
border: 1px solid #0640ff;
padding: 5px;
background-color: rgb(255, 255, 255);
}
#lineEsit{
font: bold 15px;
border-radius: 8px;
border: 1px solid #0640ff;
padding: 5px;

}
QPushButton#pushButton{
background-color: black;
color: white;
font: bold 15px;
border-radius: 10px;
transition: 0.2s;
}
QPushButton#pushButton:hover {
background-color: rgb(80,80,80);
font: bold 17px;
}
QPushButton#pushButton:pressed {
background-color: rgb(50,50,50);
}

QPushButton#pushButton:focus{
box-shadow: 0 0 5px rgba(0,0,255,0.5);
font: bold 15px;
Width: 150px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>918</width>
     <height>512</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>401</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>5</pointsize>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="toolTipDuration">
     <number>16</number>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="locale">
     <locale language="Duala" country="Cameroon"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string notr="true">Enter your school or college name</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2010</number>
     </property>
     <property name="maximum">
      <number>2024</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2020</number>
     </property>
     <property name="maximum">
      <number>2030</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(54, 66, 51);</string>
     </property>
     <property name="text">
      <string>School or Cpllege / University :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(54, 66, 51);</string>
     </property>
     <property name="text">
      <string>Start year :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>140</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(54, 66, 51);</string>
     </property>
     <property name="text">
      <string>End year (or expected) :</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 15px;
border-radius: 8px;
border: 1px solid #0640ff;
padding: 5px;
</string>
    </property>
    <property name="placeholderText">
     <string>Please enter your address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 15px;
border-radius: 8px;
border: 1px solid #0640ff;
padding: 5px;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="placeholderText">
     <string>Enter recent job title </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: bold 15px;
border-radius: 8px;
border: 1px solid #0640ff;
padding: 5px;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="placeholderText">
     <string>Enter employment type</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 15px;
border-radius: 8px;
border: 1px solid #0640ff;
padding: 5px;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="placeholderText">
     <string>Enter recent company</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 66, 51);</string>
    </property>
    <property name="text">
     <string>Most recent job title :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 66, 51);</string>
    </property>
    <property name="text">
     <string>Employment type :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 66, 51);</string>
    </property>
    <property name="text">
     <string>Most recent company :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 66, 51);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Birth date :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 66, 51);</string>
    </property>
    <property name="text">
     <string>Address :</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Magneto</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(54, 66, 51);
color: white;
padding: 5px;
border-top-left-radius: 40px;
border-bottom-right-radius: 40px;
</string>
    </property>
    <property name="text">
     <string>PORJOB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>440</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>918</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar_4">
   <property name="windowTitle">
    <string>toolBar_4</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
